lib/pages/stage_details_page.rb

```diff
@@ -83,6 +85,26 @@ def check_pipeline_status(status)
       page_status_bar.find('.run_results').find('.result').text.include?(status)
     end
 
+    def verify_status_with_job(status,job)
+      jobs_passed.text.include?status  and is_job_in_passed_list?(job)         
+    end      
+  
+
+    def is_job_in_passed_list?(job)
+      passed_job_list.each{ |jobs| 
+        return true if jobs.text.include?job
+       }  
+    end
+
+    def  verify_stage_trigger_is_enabled?(stage)
+      page.has_selector?("#stage_bar_trigger_#{stage}")
+    end  
+
+    def verify_stage_has_no_actions(stage)
+      binding.pry
```

Comment:
Please remove these debug steps

Reply:
Also this file doesnt seems to be properly formatted. Please do a document format